shell/platform/tizen/tizen_window_elementary.h

```diff
@@ -65,13 +63,14 @@ class TizenWindowElementary : public TizenWindow {
 
   void UnregisterEventHandlers();
 
+  void PrepareInputMethod();
```

Comment:
I haven't decided on this yet. because I was considering #292, I was planning to pick a suitable location for this after #291 and #292 merged. 

Can I do that? or should I add `TizenWindow` implementation file now?

Reply:
Okay, no problem!